client/app/dbaas/logs/detail/dashboards/logs-dashboards.html

```diff
@@ -7,11 +7,16 @@
         (<logs-upgrade-quota-link></logs-upgrade-quota-link>)
     </p>
 
-    <button type="button" class="oui-button oui-button_secondary"
-        data-ui-sref="dbaas.logs.detail.dashboards.add"
-        data-ng-disabled="ctrl.delete.loading">
-            <i class="oui-icon oui-icon-add pr-1" aria-hidden="true"></i> {{ ::'logs_dashboards_add' | translate }}
-    </button>
+    <div class="mt-4 mb-2">
+        <logs-add-tool
+            data-ng-if="!ctrl.quota.loading"
+            text = "{{ ::'logs_dashboards_add' | translate }}"
```

Comment:
Copy-Paste trivial issue. Please remove the spaces between attribute and '=' and the content

Reply:
Spaces removed in all places where the component is used